src/afl-fuzz.c

```diff
@@ -167,8 +167,8 @@ static void usage(u8 *argv0, int more_help) {
       "                  See docs/README.MOpt.md\n"
       "  -c program    - enable CmpLog by specifying a binary compiled for "
       "it.\n"
-      "                  if using QEMU/FRIDA or if you the fuzzing target is "
-      "compiled"
+      "                  if using QEMU/FRIDA or the fuzzing target is "
+      "compiled\n"
```

Comment:
Wouldn't the terminal automatically wrap this line?

Reply:
On Mac terminal
<img width="1439" alt="Screen Shot 2022-07-24 at 6 41 18 PM" src="https://user-images.githubusercontent.com/5111198/180643600-fd448480-dabc-48b1-8acd-c17961e96850.png">